index.js

```diff
@@ -1,73 +1,82 @@
-var async = require('async');
+module.exports = Sevnup;
 
-var VNodeStore = require('./lib/vnode-store.js');
+var _ = require('lodash');
+var crypto = require('crypto');
+var async = require('async');
 
-//TODO (joseph@): Config.
-var TOTAL_VNODES = 14;
-var MAX_PARALLEL_TASKS = 5;
+var DEFAULT_TOTAL_VNODES = 14;
```

Comment:
we have to bump this up. it's actually dangerous to defualt this low. would be error prone.